docs/_docs/Starters Academy (LDSSA)/06-Remote-Experience.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Remote Experience
+category: Starters Academy (LDSSA)
+order: 6
+---
+
+
+## Remote, safe, and still a community
+
+As everyone is aware, we are living through the biggest public health crisis in living memory. 
+In the interest of safeguarding the instructors and students physical health, we are doing this 
+batch 100% remote.
+
+In normal times, the course is 90% remote. Of the 10% of the time that we meet up, the bootcamp 
+is the most important. It is where everyone spends a few days together and the community aspect 
+of the LDSSA takes root. We think that this sense of community that you get from meeting people 
+in person is what makes the difference between success and dropout for many students. Most online 
+only courses have very high dropout rates (~90%). Our dropout rate is historically very low (~10%).
+
+So our challenge here is to find a way to maintain a sense of community while staying safe.
+
+## Different personalities, different experiences
+
+Everyone is different. Some people really like socializing. Some people don't like people. 
+Everything in between exists. One interesting thing about this batch is going to be that you 
+have much more control over the type of experience you have. In previous batches, we had in 
+person events that we required attendance to. This created an environment where social was 
+the default. Now it will be the opposite where isolated is the default. That's fine for 
+introverts and not so great for extroverts.
+
+Both experiences will use the same technology. The difference is how you use them. So let's 
+talk first about the tech and then how you can use these tools to craft the type of experience 
+that you want.
+
+## Your experience
+
+There are a few different types of experiences that you can have with this format. A social 
+one or a solitary one. The choice is yours but in the case that you are feeling lost or demotivated, 
+we strongly recommend the social experience. This is how you make the friends that will help you 
+through the course.
+
+### Experience 1: Solitary
+
+This one is easy. If you are overqualified for the course, are super confident that you can 
+breeze through all of the material, and are not a particularly social person then this is the 
+one for you. You leave the auditorium room open, pay attention when presentations are up, and 
+then just work away for the rest of the time.
+
+If you are an introverted person that is not particularly confident in your ability to complete 
+the course, the solitary experience will be very tempting. DON'T DO IT! This is a trap. It's a 
+nice, warm, safe, familiar blanket that will suffocate your chances of completing the course. 
+This is how you end up in your own head with a nice piece of imposter syndrome eating away at 
+your path to success. Bite the bullet and go for Experience 2.
+
+
+### Experience 2: Social
+
```

Comment:
As someone who struggled with taking the initiative on Batch 3, I would suggest a slight change here, since this might be a bit too "aggressive" for shy people. :'D

I would suggest something along the lines of "reach out to Student Success team", since one of the main goals of Student Success is to make people feel welcome and adapt to the group.

So instead of us telling them to reach out and do something that-s like bungee jumping for a shy person, I would suggest kindly pushing them towards the buddy people, aka Student Success folks, since its the purpose of this team to make students have a welcoming and smooth experience. It-s usually easier to reach out to a person whose job is mainly to make you feel welcome, than straight towards the other unknown students (people that might welcome you, and people that might not, even if inadvertently).

Reply:
Only after that smoother intro, would I leave the rest of the section. :)

Reply:
Maybe student success wants to pick that, if they are putting themselves are the primary points of contact here :)

Reply:
@rjna

Reply:
We can finish this PR and leave the decision for @rjna to be done on a new PR